firmware/integration/rusefi_config.txt

```diff
@@ -1007,10 +1034,14 @@ float[CRANKING_ADVANCE_CURVE_SIZE] crankingAdvance    ;Optional timing advance t
 	int16_t coastingFuelCutRpmHigh;+This sets the RPM limit above which the fuel cut is deactivated, activating this maintains fuel flow at high RPM to help cool pistons;"rpm",        1,     0,  0,    5000,  0
 	int16_t coastingFuelCutRpmLow;+This sets the RPM limit below which the fuel cut is deactivated, this prevents jerking or issues transitioning to idle;"rpm",        1,     0,  0,    5000,  0
 	int16_t coastingFuelCutTps;+percent between 0 and 100 below which the fuel cut is deactivated, this helps low speed drivability.;"%",        1,     0,  0,    100,  1
-	int16_t coastingFuelCutClt;+Fuel cutoff is deactivated below this coolant threashold.;"C",        1,     0,  -100,    100,  0
+	int16_t coastingFuelCutClt;+Fuel cutoff is deactivated below this coolant threshold.;"C",        1,     0,  -100,    100,  0
 	
 	int16_t pidExtraForLowRpm;+Increases PID reaction for RPM<target by adding extra percent to PID-error;"%",        1,     0,  0,    100,  0
-	int16_t coastingFuelCutMap;+MAP value above which coasting fuel cut is dissabled to aid piston cooling;"kPa", 1, 0, 0, 250, 0
+<<<<<<< HEAD
+	int16_t coastingFuelCutMap;+MAP value above which coasting fuel cut is disabled to aid piston cooling;"kPa", 1, 0, 0, 250, 0
+=======
+	int16_t coastingFuelCutMap;+MAP value above which coasting fuel cut is disabled to aid piston cooling;"kPa", 1, 0, 0, 250, 0
+>>>>>>> ff39a7263338c337357e825c2b235b60c54dea0c
```

Comment:
last one

Reply:
Also, this isn't what this setting is for.  This is the MAP at which we reintroduce fuel (ie, you put your foot back on the pedal)